src/pages/Location/LocationConfirmation/LocationConfirmation.js

```diff
@@ -46,9 +71,7 @@ const LocationConfirmation = () => {
                 <div className="col-12 pt-2">
                   <Typography variant="body1">
                     Your cover crop recommendations will come from the Plant Hardiness Zone
-                    {' '}
```

Comment:
these spaces are needed otherwise it says `Your cover crop recommendations will come from the Plant Hardiness Zone7NECCC dataset`